### PR TITLE
Task estimation in scrum diya

### PR DESCRIPTION
**Pull request by diyarenny**
Please review my PR and give back feedbacks. This PR contains the following sub sections:
 - Challenges
 -  Overcoming Estimation Challenges
 - Bad Practices of Task Estimation
 - Advantages and Disadvantages

### CODE REVIEW

**Review by JouharaAlwardi — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,74 @@
...
+
+---
+
+Teams can get better at estimating tasks to provide accurate forecasts. Common biggest pain point for the teams is predictability. The following scenarios are examples where the team may fail to conduct good task estimation practices :
+
+
+1. Teams are interrupted with production issues or other 
+responsibilities.
+
+2. Teams gives estimates as if they were full-time dedicated, but they have many initiatives in progress.
```
Comment (xtreger) [inline README.md]: Small spelling mistake here.

**Review by xtreger — CHANGES_REQUESTED** [review]
Looks good, just one mistake that needs a fix.

File: README.md (hunk 1 above)
Comment (diyarenny) [reply README.md:44]: Where is the spelling mistake?

Comment (diyarenny) [reply README.md]: > Small spelling mistake here.

Fixed the spelling

**Review by xtreger — APPROVED** [review]
Looks good, a nice fix of the image source, approving PR.

**Pull request merged**